contracts/proposers/BurnAuction.sol

```diff
@@ -0,0 +1,101 @@
+pragma solidity ^0.5.15;
+import { Chooser } from "./Chooser.sol";
+import { Bitmap } from "../libs/Bitmap.sol";
+import { Ownable } from "@openzeppelin/contracts/ownership/Ownable.sol";
+
+contract BurnAuction is Chooser, Ownable {
+    uint32 constant BLOCKS_PER_SLOT = 100;
+    uint32 constant DELTA_BLOCKS_INITIAL_SLOT = 1000;
```

Comment:
We now have 2 new parameters here. And would 100 blocks per batch makes our tps 100x less?

Reply:
I dont think so, am I missing something? Can a operator push 1 batch per slot only?

Reply:
[winner can propose as many batches they want] so no harms to tps